en/about/index.md

```diff
@@ -8,10 +8,15 @@ image: "/banner/aboutus.jpg"
 ---
 
 ## SOCIAL MISSION
-The club Walking Machine is before all a way to learn. Through their participation in the activities of the club, members acquire knowledge in the fields of engineering related to their. The nature of the competition in which the club participated, It has an important social mission: assistance to persons with reduced mobility. It is developing robots capable of performing simple household tasks that we want to improve the lot of our elders. On another level, the contribution of the youth club is just as important. With various events for the promotion of Robotics, an interest to continue in the area is found in young people in college and secondary schools.
+
+The Walking Machine club is, before everything, a way to learn. Its members gain knowledge through practice in the different fields of ingeeneering. The scientific club has also a social mission, to help citizen with reduced mobility, due to the nature of the competition in which it participates. By developping robots able to accomplish simple domestic tasks, we seek to make life easier for our elders. On another level, we also want to inspire youth, by showing up to diverse promotional events in highschool and colleges. 
 
 ## PHILOSOPHY
-The club has a very important philosophy of openness. Indeed, any tool or free software is recommended when technical options. This choice is motivated by the hope of universal sharing of knowledge. Interdisciplinary collaboration is also strongly encouraged. This leads to develop more knowledge and brings a knowledge of related fields which, otherwise, would that inaccessible. The members are not separated by discipline, but instead distributed according to different projects. All ideas are considered.
+
+The clubs philosophy is being open to give back to the community. In fact, we will use free software before using proprietary software to accomplish our goals. This choice is motivated by the hope of sharing universal knowledge. We are also a multidisciplinary team with a divers background. This helps us build the best robot possible using a multitude of technologies. This helps to developp knowledge to the largest amount of people in connex fields, which would otherwise be almost unaccesible. Our team members are not separated by disciplins, but by projects, and everyone's ideas are considered. 
```

Comment:
The **club's** philosophy is **all about open mindedness.**
We are also a multidisciplinary team with a **diverse** background. This helps us build the best robot possible using a multitude of technologies **and help the largest amount of members to develop** knowledge in **multiple** fields, which would otherwise be almost **unaccessible.** Our team members are not separated by **disciplines,** but by projects, and everyone's ideas are considered.